core/modules/filters/x-listops.js

```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
         return results;
     };
 
+    /*
+    Return the number the str represents
+    Return default if it's not a number
+    */
+    function getInt( str, deflt ) {
+    	var i= parseInt( str );
+    	return isNaN(i) ? deflt : i;
+    }
+
     /*
     Moves a number of items from the tail of the current list before the item named in the operand
     */
     exports.putbefore = function (source, operator) {
         var results = prepare_results(source),
             index = results.indexOf(operator.operand),
-            count = parseInt(operator.suffix) || 1;
+            count = getInt(operator.suffix, 1);
```

Comment:
Again, there shouldn't be a space after the comma (and for the subsequent occurances)